src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1808,4 +1808,23 @@
   <ItemGroup Condition="'$(FeaturePortableThreadPool)' == 'true' or '$(FeatureThreadInt64PersistentCounter)' == 'true'">
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\ThreadInt64PersistentCounter.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureUtf8String)' == 'true'">
```

Comment:
Presumably this is not currently set when building mono?

Reply:
No - it is only defined here (from searching the whole repo):

https://github.com/dotnet/runtime/blob/a80e6d4c69fcfa1e17e17130c144a567b23a12a9/src/coreclr/clr.featuredefines.props#L45

Reply:
But this work should help when we want to enable Utf8String on mono (#32481).

Reply:
As far as I understand Utr8String doesn't require runtime support so it should work in mono as is (once this variable is defined)